tests/qa_automation/qa_run.pm

```diff
@@ -109,20 +108,16 @@ sub qa_script_output {
 
 # Check whether DONE file exists every $interval secs in the background
 sub wait_testrun {
-    my ($self, $interval) = @_;
```

Comment:
stick to the unpack variant - you can still use $interval //= 30; to set a default